### PR TITLE
feat(ui): Move threshold selector to set thresholds percent option

### PR DESCRIPTION
**Pull request by taylangocmen**
This updates the threshold type selector to also include comparison delta selector, also updates the threshold section ui per Alert Wizard V3 designs.

Old:
old-delt

New:
new-delt

### CODE REVIEW

File: static/app/views/alerts/incidentRules/thresholdTypeForm.tsx [hunk 1]
```diff
@@ -48,9 +55,52 @@ class ThresholdTypeForm extends PureComponent<Props> {
                   ],
                   [
                     AlertRuleComparisonType.CHANGE,
-                    hasAlertWizardV3
-                      ? 'Percent Change: {x%} higher or lower compared to previous period'
-                      : 'Percent Change',
+                    hasAlertWizardV3 ? (
+                      comparisonType === AlertRuleComparisonType.COUNT ? (
+                        'Percent Change: {x%} higher or lower compared to previous period'
```
Comment (scttcper) [inline static/app/views/alerts/incidentRules/thresholdTypeForm.tsx:60]: does this need to be translated

File: static/app/views/alerts/incidentRules/thresholdTypeForm.tsx [hunk 2]
```diff
@@ -48,9 +55,52 @@ class ThresholdTypeForm extends PureComponent<Props> {
...
+                            value={comparisonDelta}
+                            onClick={e => e.stopPropagation()}
+                            onChange={({value}) => onComparisonDeltaChange(value)}
+                            options={COMPARISON_DELTA_OPTIONS}
+                            required={comparisonType === AlertRuleComparisonType.CHANGE}
+                          />
+                        </ComparisonContainer>
+                      )
+                    ) : (
+                      'Percent Change'
```
Comment (scttcper) [inline static/app/views/alerts/incidentRules/thresholdTypeForm.tsx:102]: same here

File: static/app/views/alerts/incidentRules/thresholdTypeForm.tsx [hunk 3]
```diff
@@ -1,12 +1,15 @@
 import {Fragment, PureComponent} from 'react';
+import * as React from 'react';
```
Comment (scttcper) [inline static/app/views/alerts/incidentRules/thresholdTypeForm.tsx:2]: looks unused

**Review by scttcper — APPROVED** [review]

File: static/app/views/alerts/incidentRules/thresholdTypeForm.tsx (hunk 2 above)
Comment (taylangocmen) [reply static/app/views/alerts/incidentRules/thresholdTypeForm.tsx]: 👍

**Pull request merged**